index.html

```diff
@@ -15,255 +15,261 @@
+        <div class="desktophome">
+          <h1 class="logo">
+            Rizo<br />
+            Abdunazarov
+          </h1>
+          <p class="self">
+            I am a software developer - that is, a person who creates software
+            entirely. I help with the design or writing of the software, usually
+            I am the one who manages the project or just does the basic work.
+            Like most software developers, I am a leading software engineer.
+          </p>
+          <button class="btn" type="button">Start collaboration</button>
+        </div>
+        <img class="homeimg" src="images/landingdesk.png" alt="landingdesk" />
+      </div>
     </section>
     <section class="projects" id="projects">
-      <h2 class="project">Projects</h2>
-      <div>
-        <img
-          class="projectimages"
-          src="images/project1.png"
-          alt="project image"
-        />
-        <div class="projectcard">
-          <h3 class="projecttitle">Projects name goes here</h3>
-          <p class="languagebadge">
-            <span class="language">HTMLS/CSS</span>
-            <span class="language">Ruby on Rails</span>
-            <span class="language">JavaScript</span>
-          </p>
-          <br />
-          <button type="button" class="btn1">See this project</button>
+      <h1 class="projects-title">Projects</h1>
+      <img src="images/project1.png" width="750" class="i1" alt="project1" />
+      <div class="one projectcard">
+        <div class="thumbnail"></div>
+        <div class="project-summary">
+          <h2>Project's name goes here</h2>
+          <ul class="languagebadge">
+            <li class="language">HTML/CSS</li>
+            <li class="language">Ruby on Rails</li>
+            <li class="language">>Javascript</li>
+          </ul>
+          <button class="btn1" type="button">See this project</button>
         </div>
       </div>
-      <div>
-        <img
-          class="projectimages"
-          src="images/project2.png"
-          alt="project image"
-        />
-        <div class="projectcard">
-          <h3 class="projecttitle">Projects name goes here</h3>
-          <p class="languagebadge">
-            <span class="language">HTMLS/CSS</span>
-            <span class="language">Ruby on Rails</span>
-            <span class="language">JavaScript</span>
-          </p>
-          <br />
-          <button type="button" class="btn1">See this project</button>
+      <img src="images/project2.png" width="750" class="i2" alt="project1" />
+
+      <div class="two projectcard">
+        <div class="thumbnail"></div>
+        <div class="project-summary">
+          <h2>Project's name goes here</h2>
+          <ul class="languagebadge">
+            <li class="language">HTML/CSS</li>
+            <li class="language">Ruby on Rails</li>
+            <li class="language">>Javascript</li>
+          </ul>
+          <button class="btn1" type="button">See this project</button>
```

Comment:
- [ ] There is a space/layout issue in your showcase section because it's showing one project one above the other (like the mobile version), and these are supposed to be in a grid layout according to the Figma design. Please, kindly fix this issue to make your portfolio look even better :wink:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144728309-89624f16-a0e0-4f8c-9fe2-13cb5bde84b6.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144728313-ea5c5aa3-8e42-4559-b249-3b7f9812330f.png)